### PR TITLE
LRQA-76453 | master

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://issues.liferay.com/browse/LRQA-76453

the test will fail until the [bug](https://issues.liferay.com/browse/LPS-160064) is resolved.
tested [here](https://github.com/catarineaguiar/liferay-portal/pull/43)

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/databasepartitioning/DatabasePartitioning.testcase [hunk 1]
```diff
@@ -217,6 +217,27 @@ definition {
 			expectedCount = "1");
 	}
 
+	@description = "This is a use case for LPS-160064."
+	@priority = "3"
+	test CanAddAVirtualInstanceWithDBPartitioningEnabled {
+		property custom.properties = "virtual.hosts.valid.hosts=localhost,127.0.0.1,www.baker.com";
```
Comment (zhangyan0701) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/databasepartitioning/DatabasePartitioning.testcase:223]: Is this property necessary in this test? Looks like there is no other Database Partitioning test set up this property.

Comment (catarineaguiar) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/databasepartitioning/DatabasePartitioning.testcase:223]: In fact, it doesn't seem necessary

**Comment by catarineaguiar** [comment]
ci:test:db-partition

**Comment by catarineaguiar** [comment]
@zhangyan0701 CI test result failed as expected.

**Comment by zhangyan0701** [comment]
Thanks!

Please see https://github.com/vicnate5/liferay-portal/pull/2115.

**Pull request closed without merging**